routes/x/index.tsx

```diff
@@ -42,391 +33,242 @@ export default function ThirdPartyRegistryList({ url, data }: PageProps<Data>) {
-                            hasNext,
-                            hasPrevious,
-                            pageCount,
-                            perPage: PER_PAGE,
-                            response: data.resp,
-                            query,
-                          }}
-                        />
-                      );
-                    })()
-                    : null}
-                </div>
-              )}
-          </div>
-          <div
-            id="info"
-            class={tw`max-w-screen-xl mx-auto pt-4 pb-8 sm:pt-8 px-4 sm:px-6 lg:pt-12 lg:px-8`}
-          >
-            <dl class={tw`md:grid md:grid-cols-2 md:gap-8`}>
-              <div>
-                <div>
-                  <dt class={tw`text-lg leading-6 font-medium text-gray-900`}>
-                    How do I use modules on deno.land/x?
-                  </dt>
-                  <dd class={tw`mt-2`}>
-                    <p
-                      class={tw`text-base leading-6 text-gray-500 break-words`}
-                    >
-                      The basic format of code URLs is
-                      <InlineCode>
-                        https://deno.land/x/IDENTIFIER@VERSION/FILE_PATH
-                      </InlineCode>
-                      . If you leave out the version it will be defaulted to the
-                      most recent version released for the module.
-                    </p>
-                  </dd>
-                </div>
-                <div class={tw`mt-12`}>
-                  <dt class={tw`text-lg leading-6 font-medium text-gray-900`}>
-                    Can I find functionality built-in to Deno here?
-                  </dt>
-                  <dd class={tw`mt-2`}>
-                    <p class={tw`text-base leading-6 text-gray-500`}>
-                      No, the built-in runtime is documented on{" "}
-                      <a class={tw`link`} href="https://doc.deno.land/">
-                        deno doc
-                      </a>{" "}
-                      and in the manual. See{" "}
-                      <a href="/std" class={tw`link`}>/std</a>{" "}
-                      for the standard modules.
-                    </p>
-                  </dd>
-                </div>
-                <div class={tw`mt-12`}>
-                  <dt class={tw`text-lg leading-6 font-medium text-gray-900`}>
-                    How do I add a module to deno.land/x?
-                  </dt>
-                  <dd class={tw`mt-2`}>
-                    <p
-                      class={tw`text-base leading-6 text-gray-500 break-words`}
+          <div class={tw`border border-dark-border rounded-lg overflow-hidden`}>
+            <form
+              class={tw`px-5 py-4 flex items-center justify-between border-b border-dark-border bg-ultralight leading-none`}
+              method="get"
+            >
+              <span class={tw`font-semibold`}>Explore Modules</span>
+              <input
```

Comment:
miss "glass" icon, can we use the same search input component of the global search

Reply:
oh ok, the figma design didnt have the magnifying glass, thats why i left it out

Reply:
my mistake